config/ui-extensions/apirules/form

```diff
@@ -288,11 +288,17 @@
       widget: GenericList
       children:
         - path: '[].config'
-          name: accessStrategies.config
-          widget: KeyValuePair
+          widget: CodeEditor
```

Comment:
This CodeEditor is really ugly, but I feel this is the best we can do here to support this different formatting. 🙈